lib/guardian/topic_guardian.rb

```diff
@@ -86,13 +86,17 @@ def can_edit_topic?(topic)
 
   # Recovery Method
   def can_recover_topic?(topic)
-    topic && topic.deleted_at && topic.user && is_staff?
+    if is_staff?
+      topic && topic.deleted_at && topic.user
```

Comment:
Although this is truthy, I always prefer apis that return `true` or `false` values.